launch_testing_ros/setup.py

```diff
@@ -13,10 +13,6 @@
         ('share/launch_testing_ros/examples', glob.glob('test/examples/[!_]*.*')),
     ],
     entry_points={
-        'console_scripts': [
-            'example_talker = launch_testing_ros.examples.talker:main',
-            'example_listener = launch_testing_ros.examples.listener:main'
-        ],
```

Comment:
Why are you deleting this?

Reply:
See https://github.com/ros2/launch_ros/pull/80 for context. Those are not real examples, but a bad short term solution to a different problem.